profiling/src/profiling/uploader.rs

```diff
@@ -27,12 +29,14 @@ impl Uploader {
         receiver: Receiver<UploadMessage>,
         output_pprof: Option<Cow<'static, str>>,
         endpoint: AgentEndpoint,
+        start_time: DateTime<Utc>,
     ) -> Self {
         Self {
             fork_barrier,
             receiver,
             output_pprof,
             endpoint,
+            start_time: start_time.to_rfc3339_opts(chrono::SecondsFormat::Secs, true),
```

Comment:
The only thing I can think off where this might be wrong is if you are doing an `apachectl -k graceful` which reloads the config and could mark a new start, but technically it is still the same process. We might need to handle those cases once we add configuration to the `info.json` as it could then be changed.

Reply:
I believe this creates a new `Uploader`. If I am reading the PR correctly, this would get a value for uptime. Are you saying it _shouldn't_ because it's the same process?

I _think_ `apachectl -k graceful` should be considered to be graceful stop and start of a process so... no issue here?

Reply:
You are right, no issue here 👍 
I was thinking what would you as a user expect after a `apachectl -k graceful`, but I would expect the uptime to be reset which in this case happens.